prijateli_tree/migrations/versions/2024-01-07-0138_7a745e024c88_.py

```diff
@@ -0,0 +1,27 @@
+"""empty message
+
+Revision ID: 7a745e024c88
+Revises: e07d61e85a9d
+Create Date: 2024-01-07 01:38:16.098549
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+
+revision = "7a745e024c88"
+down_revision = "e07d61e85a9d"
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "games", sa.Column("winning_score", sa.Integer(), nullable=False)
+    )
```

Comment:
This is going to break the production database. The solution is to allow this to be nullable temporarily -- then I can write 100 in all the NULL spots in the database and then we can run a second upgrade where we set nullable to False. 

So please 

1. set nullable to TRUE here and then you can complete this pull request!  

2. Then, make a second pull request where we upgrade to a non-nullable column. (Then, you can merge that one once I let you know the database is ready.)